src/directive.js

```diff
@@ -17,10 +17,11 @@
       },
       transclude: true,
       template: '<span ng-class="{\'is-placeholder\': placeholder && !editingValue}" ng-style="{\'max-width\': \'inherit\'}" >' +
-        '<input ng-focus="onInputFocus()" ng-blur="onInputBlur()" ng-keydown="onKeyPress($event)" ' +
+        '<input ng-show="isEditing" ng-blur="onInputBlur()" ng-keydown="onKeyPress($event)" ' +
```

Comment:
See line 23 below.  The input and div are now shown conditionally, so, in order for this to work, I had to move it to the div.